pom.xml

```
@@ -97,7 +97,7 @@
     </description>
 
     <scm>
-      <connection>scm:git:ssh://git@github.com/${gitHubRepo}.git</connection>
+      <connection>scm:git:https://git@github.com/${gitHubRepo}.git</connection>
```

The username portion of the URL is not included in the plugin archetype.  Optionally may want to remove it so that the differences between the plugin and the archetype are reduced.
```suggestion
      <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
```